packages/graphql-migrations/src/migrations/SchemaMigration.ts

```diff
@@ -7,7 +7,7 @@ import { ModelChange } from '../changes/ChangeTypes';
  * @interface SchemaMigration
  */
 export interface SchemaMigration {
-  id?: string
+  id?: Number
```

Comment:
I do question this change without any motivation? 
Why we suddenly started doing it?
What issue can we link here?